src/apps/fuzzers/fuzzerCellArea.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2021 Uber Technologies, Inc.
```

Comment:
2022?

Reply:
I think I started the file in 2021 thus the header. Can be updated.